doc/funkspec.md

```diff
@@ -23,16 +23,22 @@
     * Innovatív technológiákat kell használnia, ami lehetőleg minden modern böngészőben működik
 ---
 ## 5. Jelenlegi üzleti folyamatok modellje
-
+* A jelenlegi oktatási rendszer nem használja ki a 21. századi technológiákat az oktatásban azért, hogy a tanulók számára az oktatás és a tanulás interaktívabb és reszponzívabb legyen. A diákok nagy része nem hajlandó a régi módszerek szerint tanulni ezért néhány oktatási intézmény alternatív megoldás után keres. A diákok könyvekből tanulnak és ha ellenőrizni szeretnék, hogy mennyi szó maradt meg akkor ezt otthon szokták megtenni egy külön papíron vagy nem is szokták magukat ellenőrizni. Ez rengeteg nyomtatási kiadást jelent illetve sok elhasznált lapot.
 ---
 ## 6. Igényelt üzleti folyamatok modellje
-
+* Azért hogy egyszerűbbé tegyük a diákok angol tanulását, létrehozunk egy programot ami a mai kornak megfelelő és a legtöbb böngészőn megfelelően tud megjelenni illetve futni. A diákoknak sokkal egyszerűbb lesz a tanulás hiszen a program segítségével egyszerre tanulhatnak illetve ellenőrizhetik a tudásukat addig amíg nem érzik úgy, hogy a megtanult szavak jól mennek és a tudásuk biztos. Illetve a diákoknak nem kell a könyvet elővenniük és azt átlapozva előkeresniük az adott szavakat és így ha esetleg iskolában hagyják a könyveiket akkor is tudnak tanulni. Illetve ha esetleg új szavakat szeretne megtanulni azt fel tudja vinni a rendszerbe a akár, és ez által a diákok új szavakat is fognak tudni majd tanulni.
 ---
 ## 7. Követelménylista
+|    Modul    |      ID     |           Név          |   V.   |                                                 Kifejtés                                                      |
+| :---        |    :----:   |          :---:         | :---:  |                                                  :---:                                                        |
+| Jogosultság |      K1     | Jogosultsági szintek   |  1.0   |                     Felhasználó/vendég: feladat teljesítése ; Admin: Új szavak bevitele                       |
+| Jogosultság |      K2     |       Admin felület    |  1.0   |   Felület az admin fiókkal rendelkező felhasználó számára. Tartalmaz egy felületet az új szavak beviteléhez   |
```

Comment:
Mégis lesz user login? Mert ugye fentebb írtuk hogy megszorítás, hogy ne lehessen user adatot tárolni. 
Meg ha raknánk bele logint, az elégg meg is bonyolítaná a dolgunkat xD